src/mlpack/core/data/load.hpp

```diff
@@ -411,11 +291,12 @@ bool LoadImage(const std::string& filename,
 } // namespace data
 } // namespace mlpack
 
+#include "load_impl.hpp"
```

Comment:
@shrit added that already...

Reply:
In  this case you need to move the comment from line 297 to line 289